cmd/ntt/show.go

```diff
@@ -46,7 +46,7 @@ func show(cmd *cobra.Command, args []string) error {
 			"parameters_file",
 			"hooks_file",
 			"lint_file",
-			"datadir",
+			"k3.data_dir",
```

Comment:
I find Datadir is not very useful. Maybe we should remove it?

Reply:
you are right. I will remove it. Nonetheless sub-fields will still be accessible via commands like `ntt show -- k3.data_dir`